data/src/main/java/com/aconno/sensorics/data/repository/format/FormatRepositoryImpl.kt

```diff
@@ -0,0 +1,42 @@
+package com.aconno.sensorics.data.repository.format
+
+import com.aconno.sensorics.domain.format.AdvertisementFormat
+import com.aconno.sensorics.domain.format.AdvertisementFormatJsonConverter
+import com.aconno.sensorics.domain.repository.FormatRepository
+
+class FormatRepositoryImpl(
+    private val formatDao: FormatDao,
+    private val advertisementFormatJsonConverter: AdvertisementFormatJsonConverter
+) : FormatRepository {
```

Comment:
Rename this class to indicate that this is local repository (database)

Reply:
Done.